src/validators/users.ts

```diff
@@ -0,0 +1,82 @@
+import { NextFunction, Request, Response } from 'express';
+import { check } from 'express-validator';
+
+import validateResults from '../utils/handleValidator';
+
+const userValidatorCreate = [
+  check('name')
+    .notEmpty()
+    .isString()
+    .withMessage('Name is required')
+    .isLength({ max: 20 })
+    .withMessage('Max Length 20 characters'),
+  check('userimage')
+    .notEmpty()
+    .withMessage('Image is required')
+    .isURL()
+    .withMessage('Invalid image URL'),
+  check('mail')
+    .notEmpty()
+    .withMessage('Mail is required')
+    .isEmail()
+    .withMessage('Invalid email format'),
+  check('phone')
+    .notEmpty()
+    .withMessage('Phone is required')
+    .isNumeric()
+    .withMessage('Only number is allowed'),
+  check('birthday').notEmpty().isDate().withMessage('Date is required'),
```

Comment:
on birthday we should use an specific format, for that reason this should be change as the date on bets data validator, please take a look on that file where I used and ISO to keep consistency and scalability with timezones, remember we are trying to escalate this project for other countries too if everything goes right

read this https://developer.mozilla.org/es/docs/Web/JavaScript/Reference/Global_Objects/Date/toISOString